locallib.php

```diff
@@ -103,14 +103,15 @@ function local_boostnavigation_build_custom_nodes($customnodes, navigation_node
         $nodevisible = false;
         $nodeischild = false;
         $nodekey = null;
+        $nodelanguage = null;
```

Comment:
This change is fine as I must have forgotton this line when I added the language parameter, but it isn't related to this particular feature. Could you please split it into a separate commit with a commit message like "Improve processing of the language parameter"?